lib/Bio/EnsEMBL/DataCheck/Checks/PhenotypeMultipleSeqRegions.pm

```diff
@@ -38,12 +38,17 @@ use constant {
 sub skip_tests {
   my ($self) = @_;
 
+  my $desc_dna_dba = 'Core database found';
   my $dna_dba = $self->get_dna_dba();
-  my $mca = $dna_dba->get_adaptor("MetaContainer");
-  my $division = $mca->get_division;
+  my $pass = ok(defined $dna_dba, $desc_dna_dba);
 
-  if ($division ne 'EnsemblVertebrates') {
-    return (1, "$division can have phenotypes on single seq region");
+  if ($pass) {
```

Comment:
That's true, but the code that's run in the 'tests' method executes quickly, even for human, so I thought it made for simpler code to leave it to run the test.

Reply:
Thats fine. Suggest adding a comment for later debugging or in case datacheck is used as basis of another datacheck to use core db adaptor to get meta info but runs a longer query

Reply:
Thanks Helen, that's a good suggestion.